corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -92,28 +137,66 @@ hqDefine("data_dictionary/js/data_dictionary", [
         self.showGroupPropertyTransferWarning = ko.computed(function () {
             return self.toBeDeprecated() && !deprecated && self.properties().length > 0;
         });
+
+        self.name.subscribe(changeSaveButton);
+        self.description.subscribe(changeSaveButton);
+        self.toBeDeprecated.subscribe(changeSaveButton);
```

Comment:
Another subscribe needs to be added for the `properties` field as well. This is so that when a new case property is created the save button is correctly updated.

Reply:
:thinking: This was on line 85. I wonder why I did that. Moved from line 85 to here in b0793f6d761